src/ovs/of_controller.cpp

```diff
@@ -261,6 +362,28 @@ void OFController::execute_flow(const std::string br, const std::string flow_str
     ofconn_br = NULL;
 }
 
+void OFController::execute_flow(const int br, const std::string flow_str, const std::string action) {
```

Comment:
Why these methods shown as 'newly added'? I remember they were already in 10/30 master release

Reply:
I saw there are merging conflicts for aca_main.cpp and of_controller.cpp

I suggest your first 'git rebase' current your branch with latest master branch, and then these might be solved.

Reply:
Rebased, now those functions are gone.